txnsync/bitmask_test.go

```diff
@@ -145,7 +147,9 @@ func trimIterateHelper(t *testing.T, setBits []int) {
 
 	// For types 2, let the sum exceed entries
 	if int((b)[0]) == 2 {
-		require.Equal(t, errIndexNotFound, b.iterate(setBits[len(setBits)-1], len(setBits)-1, iterfunc))
+		require.Equal(t, errIndexNotFound, b.iterate(setBits[len(setBits)-1]-1, len(setBits), iterfunc))
+		require.Nil(t, b.iterate(setBits[len(setBits)-1], len(setBits), iterfunc))
+		require.Equal(t, errDataMissing, b.iterate(setBits[len(setBits)-1], len(setBits)-1, iterfunc))
```

Comment:
This case is the same as the case on line 146:
`	require.Equal(t, errDataMissing, b.iterate(entries, len(setBits)-1, iterfunc))
`